README.md

```diff
@@ -66,45 +66,78 @@ You can find more detailed installation instructions in [installing.md](docs/get
 
 ```python
 from concrete import fhe
+import numpy as np
+
 
 def add(x, y):
     return x + y
 
+
 compiler = fhe.Compiler(add, {"x": "encrypted", "y": "encrypted"})
-inputset = [(2, 3), (0, 0), (1, 6), (7, 7), (7, 1), (3, 2), (6, 1), (1, 7), (4, 5), (5, 4)]
+max_value_included = 7
+inputset = [
+    (
+        np.random.randint(max_value_included + 1),
+        np.random.randint(max_value_included + 1),
+    )
+    for _ in range(10)
+]
 
 print(f"Compiling...")
 circuit = compiler.compile(inputset)
 
 print(f"Generating keys...")
 circuit.keygen()
 
-examples = [(3, 4), (1, 2), (7, 7), (0, 0)]
-for example in examples:
+for _ in range(4):
+
+    example = (
+        np.random.randint(max_value_included + 1),
+        np.random.randint(max_value_included + 1),
+    )
     encrypted_example = circuit.encrypt(*example)
     encrypted_result = circuit.run(encrypted_example)
     result = circuit.decrypt(encrypted_result)
     print(f"Evaluation of {' + '.join(map(str, example))} homomorphically = {result}")
+    assert result == add(*example)
 ```
 
 or if you have a simple function that you can decorate, and you don't care about explicit steps of key generation, encryption, evaluation and decryption:
 
 ```python
 from concrete import fhe
+import numpy as np
+
 
-@fhe.compiler({"x": "encrypted", "y": "encrypted"})
```

Comment:
This example meant to have the decorator.

Reply:
thx done